public/index.html

```diff
@@ -5,10 +5,18 @@
     <title>Samsung Website</title>
   </head>
   <body>
-    <h1>Samsung</h1>
-    <div id='carousel'></div>
-      <script type="text/javascript" src="http://127.0.0.1:3001/1/webpack.bundle.js"></script>
-    <div id='details'></div>
-      <script type="text/javascript" src="http://127.0.0.1:3002/1/bundle.js"></script>
-  </body>
+    <h1 style='font-size: 30px; font-family: Arial, Helvetica, sans-serif; margin-left: 5%;'>SOMESUNG</h1>
+    <div style='width: 90%; margin-left: 5%;'>
+      <div style="background-color: black; width: 100%; height: 20px; margin-bottom: 30px;"></div>
```

Comment:
If you move the h1 from line 8 inside the div at line 9, you won't need to define the margin-left on it.